src/Donations/LeftPanel/LeftPanelInfo.tsx

```diff
@@ -89,11 +92,11 @@ const LeftPanelInfo = ({
           paymentSetup={paymentSetup}
         />
       )}
-      {canShowProject && (
-        <div className={styles["project-info-container"]}>
-          <ProjectTitle projectDetails={projectDetails} />
-          {projectDetails.purpose !== "planet-cash-signup" && (
-            <ProjectInfo projectDetails={projectDetails} />
+      {canShowSummary && (
+        <div className={styles["summary-container"]}>
+          <SummaryTitle info={info} />
+          {info.purpose !== "planet-cash-signup" && (
+            <ProjectInfo projectDetails={info} />
```

Comment:
In `ProjectInfo` the property `projectDetails` is only defined as `FetchedProjectDetails` although it seems it can also be of type `PlanetCashSignupDetails` here? Is this wanted?

Reply:
Sorry, I have not seen the if condition tries to avoid using this component with info as type `PlanetCashSignupDetails`.